torch/csrc/jit/tensorexpr/cuda_codegen.cpp

```diff
@@ -718,6 +717,18 @@ class NoThreadIdxRewriter : public IRMutator {
   bool need_rewrite_ = false;
 };
 
+std::ostream& operator<<(
```

Comment:
I am not a fan of overloading "operator<<" for a generic data structure. I am perfectly fine if it is either static or within an anonymous namespace.

Reply:
I'll make this static, but I'm not sure I understand the objection?